src/kensetsu-vaults.md

```diff
@@ -0,0 +1,106 @@
+# Kensetsu Platform
+
+## Theory
+
+### Overall description of the platform
+
+Kensetsu is a lending platform with an overcollateralised stablecoin pegged to the USD. It is a MakerDAO analogue in the SORA ecosystem.
+The native token for the Kensetsu platform is the KEN token, which is used for farming and protocol rewards.
```

Comment:
There was a request to change name from `$KEN` to `$Kensetsu` but I don't know the final decision, it is not renamed yet. Better to clarify token naming.